tests/helm/kube-prometheus-stack/servicemonitors/basic-additional.input.yaml

```diff
@@ -0,0 +1,15 @@
+kube-prometheus-stack:
```

Comment:
It would be nicer if this was a Go test instead, we could avoid the output templates and just check if the servicemonitor names in the list are right.